src/helm/benchmark/scenarios/infinite_bench_sum_scenario.py

```diff
@@ -0,0 +1,72 @@
+import os
+from typing import List
+from datasets import load_dataset, Features, Value, Sequence, Dataset
+from helm.benchmark.scenarios.scenario import (
+    Scenario,
+    Instance,
+    Input,
+    Reference,
+    Output,
+    CORRECT_TAG,
+    TEST_SPLIT,
+)
+from helm.common.general import ensure_directory_exists
+
+
+class InfiniteBenchSumScenario(Scenario):
+    """InfiniteBenchSum
```

Comment:
Is there a space between InfiniteBench and Sum?

Reply:
Yes, corrected